lib/solidus_related_products/factories.rb

```diff
@@ -26,6 +30,10 @@
       association :relation_type, factory: [:relation_type, :from_variant_to_variant]
     end
 
+    before :create do |relation, options|
+      relation.relation_type.update(bidirectional: options.bidirectional)
```

Comment:
Can you use `update!` so that the factory fails loudly if something goes wrong?

Reply:
👍